draft-packages/illustration/KaizenDraft/Illustration/Scene.tsx

```diff
@@ -6,22 +6,40 @@ export type SceneProps = Pick<BaseProps, "alt" | "classNameAndIHaveSpokenToDST">
 export const EmptyStatesAction = (props: SceneProps) => (
   <Base {...props} name="illustrations/scene/empty-states-action.svg" />
 )
+export const EmptyStatesActionHeart = (props: SceneProps) => (
+  <Base {...props} name="illustrations/heart/scene/empty-states-action.svg" />
+)
 
 export const EmptyStatesInformative = (props: SceneProps) => (
   <Base {...props} name="illustrations/scene/empty-states-informative.svg" />
 )
+export const EmptyStatesInformativeHeart = (props: SceneProps) => (
+  <Base
+    {...props}
+    name="illustrations/heart/scene/empty-states-informative.svg"
+  />
+)
 
 export const EmptyStatesNegative = (props: SceneProps) => (
   <Base {...props} name="illustrations/scene/empty-states-negative.svg" />
 )
+export const EmptyStatesNegativeHeart = (props: SceneProps) => (
+  <Base {...props} name="illustrations/heart/scene/empty-states-negative.svg" />
+)
 
 export const EmptyStatesPositive = (props: SceneProps) => (
   <Base {...props} name="illustrations/scene/empty-states-positive.svg" />
 )
+export const EmptyStatesPositiveHeart = (props: SceneProps) => (
+  <Base {...props} name="illustrations/heart/scene/empty-states-positive.svg" />
+)
```

Comment:
I was thinking something along the lines of...
```
export const EmptyStatesPositiveHeart = (props: SceneProps) => {
   const theme = const theme = useTheme(); 
   if (theme.themeKey === "zen") {
      // Old illustration
      return <Base {...props} name="illustrations/scene/empty-states-positive.svg" />
   }
   // new Heart illustration
   return <Base {...props} name="illustrations/heart/scene/empty-states-positive.svg" />
}
```

My rationale for this is these Illustrations map well from the old ones to the new ones:

<img width="893" alt="Screen Shot 2021-03-24 at 4 59 24 pm" src="https://user-images.githubusercontent.com/1621182/112262783-50148c00-8cc2-11eb-85cb-88c4ff35d9a4.png">
<img width="938" alt="Screen Shot 2021-03-24 at 4 59 17 pm" src="https://user-images.githubusercontent.com/1621182/112262792-530f7c80-8cc2-11eb-8cc3-457bd5b2140f.png">

Reply:
good idea i'll do that